src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -246,6 +246,42 @@ Environment variables:
 		"only print pachctl's version, but don't make any RPCs to pachd. Useful "+
 		"if pachd is unavailable")
 
+	clusterIDCmd := &cobra.Command{
+		Use:   "cluster-info",
+		Short: "Return cluster information.",
+		Long:  "Return cluster information.",
+		Run: cmdutil.RunFixedArgs(0, func(args []string) (retErr error) {
+			if !noMetrics {
+				start := time.Now()
+				startMetricsWait := metrics.StartReportAndFlushUserAction("ClusterInfo", start)
+				defer startMetricsWait()
+				defer func() {
+					finishMetricsWait := metrics.FinishReportAndFlushUserAction("ClusterInfo", retErr, start)
+					finishMetricsWait()
+				}()
+			}
```

Comment:
I was under the impression this snippet of code was fairly specific to `version` since we expect that to be the first command that people run. Is it needed here too? I don't think we have it on most other commands.

Reply:
Yup, I removed this